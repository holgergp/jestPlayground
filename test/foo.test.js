const foo = require('../src/foo');

test('renders correctly', () => {
  expect(foo.renderSomething()).toMatchSnapshot();
});