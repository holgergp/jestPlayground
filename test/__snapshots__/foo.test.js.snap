exports[`test renders correctly 1`] = `"My rendered String changed"`;
